### PR TITLE
Ruby snippets for account api

### PR DESCRIPTION
**Pull request by KBbitsP**

### CODE REVIEW

File: source/includes/_account.md [hunk 1]
```diff
@@ -620,16 +640,18 @@ Accounts allAccounts = accountApi.getAccounts(offset,
...
-account.find_in_batches(offset,
-                        limit,
-                        with_balance,
-                        with_balance_and_cba,
-                        options)
+allAccounts = account.find_in_batches( offset,
+                                    limit,
+                                    with_balance,
+                                    with_balance_and_cba,
+                                    options)
```
Comment (reshmabidikar) [inline source/includes/_account.md:654]: Are there some extra spaces here?

File: source/includes/_account.md [hunk 2]
```diff
@@ -760,18 +782,20 @@ List<Account> accountsByKey = accountApi.searchAccounts(searchKey,
...
-                                      limit,
-                                      with_balance,
-                                      with_balance_and_cba,
-                                      options)
+accounts = account.find_in_batches_by_search_key( search_key,
+                                                  offset,
+                                                  limit,
+                                                  with_balance,
+                                                  with_balance_and_cba,
+                                                  options)
```
Comment (reshmabidikar) [inline source/includes/_account.md:798]: same comment as above, if there are extra spaces, can you please delete them?

File: source/includes/_account.md [hunk 3]
```diff
@@ -2578,12 +2658,14 @@ List<PaymentMethod> paymentMethods = accountApi.getPaymentMethodsForAccount(acco
...
+
+account_id = "faf239a5-456a-4eb9-aef9-8d2254ef57dc"
 with_plugin_info = false
 
-payment_method.find_all_by_account_id(account_id, 
-                                      with_plugin_info,
-                                      options)
+accountPaymentMethods  = payment_method.find_all_by_account_id(account_id, 
+                                                                with_plugin_info,
+                                                                options)
```
Comment (reshmabidikar) [inline source/includes/_account.md:2668]: Additional spaces again?

**Review by reshmabidikar — APPROVED** [review]
Looks good, I just noticed that there are some space/newline characters present in some of the code snippets. I'm not sure if these will cause an issue. Could you please take a look and make corrections if required?

File: source/includes/_account.md (hunk 2 above)
Comment (KBbitsP) [reply source/includes/_account.md:798]: Hi Reshma, Yes I added these to align all the values. Below are local render and editor screenshots : 

image
image

**Pull request merged**